x-pack/plugins/observability_solution/observability_onboarding/public/application/experimental_onboarding_flow.tsx

```diff
@@ -29,6 +30,9 @@ import { CustomLogsPanel } from './quickstart_flows/custom_logs';
 export function ExperimentalOnboardingFlow() {
   const history = useHistory();
 
+  const [queryParams] = useSearchParams();
+  const queryString = queryParams.size ? `?${queryParams.toString()}` : '';
```

Comment:
I believe `location.search` either from `useLocation()` or just `window.location` would produce the same thing here